submissions/Mifaresss/js_dom/script.js

```diff
@@ -0,0 +1,126 @@
+	},
+	{
+		id: 9,
+		header: 'Черный клевер',
+		img: './images/black-clever.jpg',
+		subHeader: 'Краткое описание',
+		description: `Их свела общая судьба. Аста и Юно – двое сирот, мечтающих найти своё место под солнцем и когда-нибудь прославиться как великие маги. Однако общая судьба - не общий дар, и, как это часто бывает, пока один достигает определённых успехов, другой вынужден плестись позади, не проявляя практически никакого таланта. Но друг, как известно, познаётся в беде. Когда Юно, получившему могущественный гримуар четырёхлистного клевера, угрожает опасность, Аста раскрывает весь свой потенциал и получает гримуар, на котором красуется клевер с пятью лепестками, прозванный «Чёрным».`
+	},
+];
+
+const cardsList = document.querySelector('.main__content');
+
+function createCards(id, header, img, subHeader, description) {
+	const cardsHTML = `
+		<li class="main__content-item ${id}">
+			<h2 class="sub-title">${header}</h2>
+			<img src="${img}" alt="${header}" class="main__img">
+			<h3 class="title-description">${subHeader}</h3>
+			<p class="title-description__text">${description}</p>
+		</li>
+		`
+	cardsList.innerHTML = cardsHTML;
+}
+
+function showCard(event) {
+	if (event.target.tagName !== 'BUTTON') return false;
+	const filterClass = event.target.dataset.f;
+	const { id, header, img, subHeader, description } = cards.find(card => card.id == filterClass);
+	createCards(id, header, img, subHeader, description);
+	const homeScreen = document.querySelector('.home-screen');
+	homeScreen.classList.add('display-none');
+}
+function hidingBurger(event) {
+	if (event.target.tagName !== 'BUTTON') return false;
+	if (iconMenu.classList.contains('_active')) {
+		iconMenu.classList.remove('_active');
+		buttons.classList.remove('_active');
+		document.body.classList.remove('_lock');
+	}
+}
+
+const buttons = document.querySelector('nav');
+buttons.addEventListener('click', hidingBurger);
+buttons.addEventListener('click', showCard);
+
+const iconMenu = document.querySelector('.icon-menu');
+iconMenu.addEventListener('click', e => {
+	iconMenu.classList.toggle('_active');
+	buttons.classList.toggle('_active');
+	document.body.classList.toggle('_lock');
+});
+
+const homeScreenDescription = document.querySelector('.home-screen__description');
+function changeValueParagraph() {
+	if (document.documentElement.clientWidth < 850) {
+		homeScreenDescription.innerHTML = 'Кликай на значок меню справа вверху!)'
+	}

```

Comment:
Realize this by rules in the CSS.

Reply:
Initially, I wanted to do this, but did not know how. Now for some reason I immediately remembered pseudo-elements!) "With a fresh mind")))

Reply:
done